README.md

```diff
@@ -59,11 +59,29 @@ To add a gallery image to a notebook, select a cell with an output image and add
 
 In addition to serving up standalone notebooks, this repository also includes the infrastructure needed to build the tutorials into HTML documentation using [Sphinx](https://www.sphinx-doc.org/).  Along with the Qiskit dependencies, building the documentation requires the following:
 
-```bash
-pip install Sphinx
-pip install sphinx-rtd-theme
-pip install nbsphinx
-```
+1) Fork and clone the repository. 
+
+2) Create a new virtual environment and install pip:
+      ```bash
```

Comment:
@bopardikarsoham I think this is my fault, but can you unify the spaces in front of the code cells for each step? After this it should be good to merge!

Reply:
Fixed in https://github.com/Qiskit/qiskit-tutorials/pull/1332/commits/a1e19711780e1dfcff90fc763b9fbc83bc980728